lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -970,6 +1068,127 @@ sub get_dataset_allele_response {
   return $ds_response;
 }
 
+sub get_vep_frequency {
+  my $vep_consequences = shift;
+
+  my @gnomad_frequency;
+
+  foreach my $consequence (@{$vep_consequences}) {
+    foreach my $variant (@{$consequence->{colocated_variants}}) {
+      foreach my $allele (keys %{$variant->{frequencies}}) {
+        my $frequencies = $variant->{frequencies}->{$allele};
+        foreach my $key (keys %{$frequencies}) {
+          if($key =~ /gnomad/) {
+            my ($gnomad_type, $pop) = $key =~ /(.*)_(.*)/;
+            my $freq_obj;
+            $freq_obj->{alleleFrequency} = $frequencies->{$key};
+            $freq_obj->{allele} = $allele;
```

Comment:
`allele` is not a property of **Frequencies**:
* http://docs.genomebeacons.org/schemas-md/obj/frequencies/
* https://github.com/ga4gh-beacon/beacon-v2/blob/e7d79620cccc3c93e9a559d9add3955491a6614c/docs/schemas-md/genomicVariations_defaultSchema.md?plain=1#L92-L105

Reply:
That's true. I added it because I think an allele is needed for each frequency in case there are multiple alt alleles.
I'm going to suggest an update for the Beacon schema.